### ISSUE TITLE
Problem Memcached

### ISSUE THREAD

**Issue by alby696** [WhatCD/Gazelle#120]
Hello to all,
installed everything correctly without errors, edited the config.php and with the correct path together with the door and edited the memcached.conf file, restarted the service but when I open the index.php page I always have "Memcache Extension not loaded."

![schermata 2018-05-10 alle 09 06 58](https://user-images.githubusercontent.com/15062741/39856820-958f50d4-5431-11e8-9686-a79aade51638.png)

Does anyone have any suggestions to give me?

Thank you
alby696

**Comment by scriptzteam** [comment]
do you have php-memcache ?

**Comment by alby696** [comment]
Yes, i have install memcached and php-memcached but the problem persist.

**Comment by alby696** [comment]
Now i have remove memcached e php-memcached and reinstall memcached and php-memcache but now i have a white page.

**Comment by scriptzteam** [comment]
Load few times the index.php file on web server then do this:

`cd /var/log/apache2/`

`cat error.log`

or 

`tail -f error.log`

You can see if any error occurs

Probably try do enable php error log :)

P.S.: DID YOU ENABLED THE MEMCACHE VIA `phpenmod `??

**Comment by alby696** [comment]
The apache directory does not exist as a log because it is not installed with nginx installed.
I have enabled the php log but it does not say anything except that the configuration file is correct.
The memcache is enabled in php because if I do the phpinfo code it comes out to me that it is installed.

![schermata 2018-05-13 alle 21 33 40](https://user-images.githubusercontent.com/15062741/39971065-7262743a-56f5-11e8-9b35-37d81de87a43.png)

**Comment by scriptzteam** [comment]
Ah sry nginx right xD

Then look at nginx log folder :)

`cd /var/log/nginx/`

And check error logs...

Other than that, idk since for me gazelle works :)

**Comment by alby696** [comment]
thanks so much....

**Comment by scriptzteam** [comment]
:)

**Comment by miitchel** [comment]
for me it doesnt give any error logs, it just says "Memcache Extension not loaded.", while in phpinfo() it does show memcached

**Comment by scriptzteam** [comment]
enable error logging.
is memcached installed? i mean not for php for just memcached
